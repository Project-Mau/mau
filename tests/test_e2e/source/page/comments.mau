// This is a comment and will not appear in the output.

////
This is a multiline
comment.
All these lines will disappear.
////

