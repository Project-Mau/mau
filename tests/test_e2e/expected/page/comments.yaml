data:
  args: []
  content: []
  kwargs: {}
  subtype: null
  tags: []
  type: document

